### PR TITLE
add the install subcommand in the makefile

### PR DESCRIPTION
**Pull request by hellosmallstone**
install subcommand will copy the binary to the local GOBIN

Signed-off-by: wuxu <wuxu1103@163.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -53,3 +53,8 @@ unit-test:
 integration-test:
 	@echo "Run integration test......"
 	./test/integration-test.sh $(FABRIC_VERSION) $(INTERGATION_CASE)
+
+.PHONY: install
```
Comment (SamYuan1990) [inline Makefile:57]: if `./tape` not exist?

**Review by SamYuan1990 — CHANGES_REQUESTED** [review]

**Review by SamYuan1990 — DISMISSED** [review]

**Comment by hellosmallstone** [comment]
I don't know why the GOBIN is empty in the CI environment. So I added the judgement of whether GOBIN is empty.

File: Makefile [hunk 2]
```diff
@@ -53,3 +54,12 @@ unit-test:
...
 	@echo "Run integration test......"
 	./test/integration-test.sh $(FABRIC_VERSION) $(INTERGATION_CASE)
+
+.PHONY: install
+install: tape
+	@echo "Install tape to GOBIN......"
+ifneq ($(BINPATH),)
+	cp ./tape $(BINPATH)
+else
+	@echo "Warning: GOBIN is not configured!"
```
Comment (SamYuan1990) [inline Makefile:64]: @echo "Warning: GOBIN is not configured! You are able to copy tape binary file to your $PATH" 

这类描述？我不确定windows下是不是$PATH.
另外考虑下755这样的文件权限？

**Review by SamYuan1990 — DISMISSED** [review]

File: Makefile (hunk 2 above)
Comment (guoger) [reply Makefile:64]: 应该可以跑`go install`吧？

Comment (guoger) [reply Makefile:64]: e.g.:
```make
.PHONY: install
install:
	@echo "Install tape"
	go install -tags "$(GO_TAGS)" -ldflags "$(GO_LDFLAGS)" ./cmd/tape
```

**Comment by guoger** [comment]
also, we should add `make basic-check` to run gofmt and linter, for example: https://github.com/kubernetes/kubernetes/blob/master/hack/verify-gofmt.sh and https://github.com/hyperledger/fabric/blob/bbdb4e7929e4ed32024a42d7593e4f4db5fbc39a/Makefile#L109

**Review by guoger — APPROVED** [review]
`basic-checks` should be extracted to its own job, preferably executed **before** everything else

**Pull request merged**